applications/IOIOManager/src/main/java/ioio/manager/FileReturner.java

```diff
@@ -31,6 +31,6 @@
 import androidx.appcompat.app.AppCompatActivity;
 
 public interface FileReturner {
-	String ERROR_MESSAGE_EXTRA  = "ioio.manager.ERROR_MESSAGE";
-	int RESULT_ERROR = AppCompatActivity.RESULT_FIRST_USER;
+    String ERROR_MESSAGE_EXTRA = "ioio.manager.ERROR_MESSAGE";
```

Comment:
This change is just noise, in one comit a wrong format and in the other format back.

Do you know these commands ? 
`git commit --fixup <some SHA1>`
and later to squash all once
`git rebase -i master --autosquash`

It's exactly for such a workflow to have a beautiful git history.

Reply:
Sorry, I don't think I did that intentionally. It must have been an auto formatting issue with Android Studio. I've seen those pop up from time to time in my commits after mistakenly including them in a `git add .`. This seems to arise from differences in .idea/codeStyles that I never manually change. I will try to take more care in catching these before they are included in my commits. 

I was unaware of that workflow for squashing. Thanks for the tip!